test/email_alert_api_test.rb

```diff
@@ -79,6 +79,48 @@
 
         assert_equal("weekly", subscription_attrs.fetch("frequency"))
       end
+
+      it "returns the subscription attributes on /latest" do
```

Comment:
This test looks like a duplicate of line 100 `it "returns the latest subscription attributes on /latest"` are they both needed?